service/src/main/java/com/codeforcommunity/processor/PublicEventsProcessorImpl.java

```diff
@@ -0,0 +1,58 @@
+package com.codeforcommunity.processor;
+
+import static org.jooq.generated.Tables.EVENTS;
+
+import com.codeforcommunity.api.IPublicEventsProcessor;
+import com.codeforcommunity.dataaccess.EventDatabaseOperations;
+import com.codeforcommunity.dto.userEvents.components.EventDetails;
+import com.codeforcommunity.dto.userEvents.responses.GetEventsResponse;
+import com.codeforcommunity.dto.userEvents.responses.SingleEventResponse;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.jooq.DSLContext;
+import org.jooq.generated.tables.pojos.Events;
+
+public class PublicEventsProcessorImpl implements IPublicEventsProcessor {
+
+  private final DSLContext db;
+  private final EventDatabaseOperations eventDatabaseOperations;
+
+  public PublicEventsProcessorImpl(DSLContext db) {
+    this.db = db;
+    this.eventDatabaseOperations = new EventDatabaseOperations(db);
+  }
+
+  @Override
+  public GetEventsResponse getEvents(List<Integer> eventIds) {
+    List<Events> e = db.selectFrom(EVENTS).where(EVENTS.ID.in(eventIds)).fetchInto(Events.class);
```

Comment:
```suggestion
    List<Events> events = db.selectFrom(EVENTS).where(EVENTS.ID.in(eventIds)).fetchInto(Events.class);
```
Even if it's clear to you, it won't always be clear to the next developer